testing/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/IsolatedProjectsParallelSyncTest.groovy

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2024 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.ide.sync
+
+
+import org.gradle.test.fixtures.server.http.BlockingHttpServer
+import org.gradle.util.internal.ToBeImplemented
+import org.junit.Rule
+
+class IsolatedProjectsParallelSyncTest extends AbstractIdeaSyncTest {
+
+    @Rule
+    BlockingHttpServer server = new BlockingHttpServer(5_000)
+
+    def setup() {
+        server.start()
+    }
+
+    @ToBeImplemented
+    def 'projects are configured in parallel during IDEA sync'() {
+        given:
+        simpleProject()
+        server.expect("configure-root")
+        server.expect("configure-a")
+        server.expect("configure-b")
+        // TODO: isolated expected behaviour for parallel sync
+//        server.expect("configure-root")
+//        server.expectConcurrent("configure-a", "configure-b")
+
+        expect:
+        ideaSync("2024.2")
```

Comment:
❔  Should we extract the version number, so it can be updated in a single place going forward?